frontend/app/src/towFactor.tsx

```diff
@@ -0,0 +1,90 @@
+import { QRCodeSVG } from 'qrcode.react'
+import React, { useRef, useState } from 'react'
+import axios from 'axios'
+import { useNavigate } from 'react-router-dom'
+import styled from 'styled-components'
+import Box from '@mui/material/Box'
+import TextField from '@mui/material/TextField'
+import Button from '@mui/material/Button'
```

Comment:
```suggestion
import { Box, TextField, Button } from '@mui/material'
```
이런 식으로 사용하는 편이 나중에 import문에서 추가/삭제할 때 좀 더 보기 편할 것 같아요